geopandas/base.py

```diff
@@ -2721,14 +2721,23 @@ def simplify(self, *args, **kwargs):
         """Returns a ``GeoSeries`` containing a simplified representation of
         each geometry.
 
+        The algorithm (Douglas-Peucker) recursively splits the original line
+        into smaller parts and connects these parts’ endpoints
+        by a straight line. Then, it removes all points whose distance
+        to the straight line is smaller than `tolerance`. It does not
+        move any points and it always preserves endpoints of
+        the original line or polygon.
         See http://shapely.readthedocs.io/en/latest/manual.html#object.simplify
         for details
 
         Parameters
         ----------
         tolerance : float
             All points in a simplified geometry will be no more than
-            `tolerance` distance from the original.
+            `tolerance` distance from the original. It has the same units
```

Comment:
> All points in a simplified geometry will be no more than `tolerance` distance from the original.

This is quite a cryptic sentence even for myself, knowing how D-P works. Since it just removes points, all points in a simplified geom are exactly 0 distance from the original. Maybe `All parts of a simplified geometry` would be better here?

As a side note, I also think that with `preserve_topology=True`, it doesn't use the exact D-P but some adaptation, pure D-P is used with `preserve_topology=False`.